src/consensus/aft/raft.h

```diff
@@ -3007,7 +3007,7 @@ namespace aft
       }
 
       snapshotter->rollback(idx);
-      store->rollback(idx, state->current_view);
+      store->rollback({get_term_internal(idx), idx}, state->current_view);
```

Comment:
The consensus, which knows about the view history, always tells the store which `TxID` to rollback to, so that further read-only transactions report the correct `TxID`, rather than the "{state->current_view}.{idx}", which is wrong.